### ISSUE TITLE
Getting very poor FPS and audio on games 

### ISSUE THREAD

**Issue by rmiller757** [WilliamLCobb/iNDS#145]
I have the latest versions of both iNDS and iOS software. I run it on my iPhone 6 and downloaded it from the builds store. Even when turning my texture packs to none I still only get a max 30fps now with a low of 15fps at times. This occurred randomly , any help ?

**Comment by essiw** [comment]
try to put auto-save on off. And set frame skip to 1. That will probably help :)

**Comment by rmiller757** [comment]
Still only 30fps with that suggestion :/ thank you though

**Comment by peter961785** [comment]
I think it could be a bug cause I experienced this on my iPhone 6 as well but now it resolved itself idk how. However when it was stuck at a max of 30fps, I didn't really experienced any noticeable lag (~60 max cores)

**Comment by essiw** [comment]
To be sure, which version of iNDS are you on (to be sure it is actually the latest) you can find that at the bottom of the preferences.

**Comment by rmiller757** [comment]
I am on 1.5.3

**Comment by DarknesGaming** [comment]
What game are you trying to run? Not all ROMs can run at full speed. Also, 30 FPS is full speed if Frame Skip is set to 1.

**Comment by rmiller757** [comment]
Pokemon diamond particularly

**Comment by DarknesGaming** [comment]
That ROM can run at full speed, but Pokemon Platinum might run faster. I have no idea why.
If you set the Frame Skip to none, you should go up to 60 FPS. Going above 60 FPS is normal starting at 1.5.3 and probably will remain normal beyond that.

**Comment by rmiller757** [comment]
Yeah that's what I'm getting at however. The frame skip is set to none, but it's still only pushing 30fps on the high end. I get as little as 10fps sometimes and this happened recently. These games used to get 60fps

**Comment by DarknesGaming** [comment]
You're not on debug mode, right? If you're only going at 30 FPS on an iPhone 6 with Frame Skip set to none, that might be the reason why.
You can check out the wiki for how to switch to release mode. I edited the wiki so that it's easier for people to figure this out.

**Comment by rmiller757** [comment]
Where's the wiki haha sorry...

**Comment by DarknesGaming** [comment]
You should look at the README.md file. The link should be there.
It helps to click on the blue words.

**Comment by aridzephyr** [comment]
I'm having the same issue. iP6, 8.4 jailbroken. Compiled straight from this repo.

Running pokemon platinum, I was getting around 50 fps, 40 after the recommended filter. When I try 2x emulation speed, the fps halves. Tried fiddling with the settings to no avail. Any help would be appreciated!

**Comment by DarknesGaming** [comment]
The fps cutting in half is normal. This happens when frame skip is enabled.

**Comment by Thecloudbacon** [comment]
I have an iPhone 4s and I'm getting 7-16 fps, what is a good solution for higher frames?

**Comment by circa-one** [comment]
Get a phone that's not 7 years old. :P

**Comment by Thecloudbacon** [comment]
i don't have $100 just lying around
